src/main/java/frc/robot/Robot.java

```diff
@@ -62,6 +71,7 @@ public void robotInit() {
    */
   @Override
   public void robotPeriodic() {
+    m_IsPracticeBot = m_BotChooser.getSelected();
```

Comment:
Alternatively if we can take the deafult option off the sendable chooser we may use the "null" case to mean "the driver hasn't picked yet" and then as soon as we finally get a value we can finish initialization. 

Also though I'm reluctant to add extra steps to the driver's job on startup. Maybe we can move this code to teleopInit that way if the driver does nothing, it will just take the competition bot as soon as you enable teleop. If the driver does change it as soon as you turn on teleop it will set itself up.